apps/challenges/admin.py

```diff
@@ -15,6 +25,11 @@
 )
 
 
+class UpdateNumOfWorkersForm(ActionForm):
+    label = " Number of Workers. (Enter a whole number while scaling. Otherwise, ignore.)"
+    num_of_tasks = forms.IntegerField(initial=-1, label=label, required=False)
```

Comment:
@KhalidRmb the checks for negative values are handled right? did you check whether all corner cases are covered?

Reply:
Only, whole numbers are allowed.